### PR TITLE
test: improve dev & testing with `src` files

### LINKED ISSUE #58: [template] yarn dev not working
Since using siroc, we need either CJS stub (wrapped by jiti/siroc) or `nuxt-ts` runtime for test fixture

### LINKED ISSUE #59: [template] test using src
We need to setup jest config to be able to test directly from src

### PR DESCRIPTION
**Pull request by danielroe**
* collect coverage from src
* uses nuxt-ts to load module src files in example fixture
* ignore an eslint error about unused variables

closes #59, closes #58

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ module.exports = {
       }
     ]
   },
+  moduleNameMapper: {
+    npm_package: '<rootDir>/src/module.ts'
```
Comment (pi0) [inline jest.config.js:12]: What if simply mapping from `src` to make code portable and clear what alias means?

File: example/nuxt.config.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
-import _module from '..'
-
 export default {
   modules: [
-    _module
-  ]
+    '../src/module.ts'
+  ],
+  watch: ['../templates/*']
```
Comment (pi0) [inline example/nuxt.config.js:5]: Doesn't it cause full nuxt restart on template change?

Comment (danielroe) [reply example/nuxt.config.js:5]:
```suggestion
```

Comment (danielroe) [inline example/nuxt.config.js:5]:
```suggestion
  ]
```

**Pull request merged**